scripts/check_release_rollout.py

```
@@ -0,0 +1,89 @@
+import urllib.error
+import urllib.request
+from typing import Any, Dict, List, Optional
+
+
+def _normalize_base_url(value: str) -> str:
+    return value.rstrip("/")
+
+
+def _fetch_release(base_url: str, timeout: float) -> Dict[str, Any]:
+    url = f"{_normalize_base_url(base_url)}/api/v1/health/release"
+    request = urllib.request.Request(url, headers={"Accept": "application/json"})
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)